packages/flags/src/libs/RemoteFlags.ts

```diff
@@ -0,0 +1,130 @@
+import urlcat from 'urlcat'
+import { has } from 'lodash-es'
+import { Flags } from './Flags.js'
+
+interface FetchResult<T extends Record<string, unknown>> {
+    flags?: T
+    timestamp: number
+}
+
+export class RemoteFlags<T extends Record<string, unknown>> extends Flags<T> {
+    private readonly KEY = 'mask-last-fetch-result'
+
+    private lastFetchResult: FetchResult<T> | null = null
+
+    private get lastStorageResult() {
+        try {
+            const json = localStorage.getItem(this.KEY)
```

Comment:
you should not store the key in the content script. it might be changed by 3rd party.

this code also does not work under MV3 background page.